Tests/ViewControllers/Products/Detail/Summary/SummaryHeaderCellControllerTests.swift

```diff
@@ -79,4 +80,14 @@ class SummaryHeaderCellControllerTests: XCTestCase {
 
         expect(self.viewController.presentedViewController is ImageViewer).to(beTrue())
     }
+
+    // MARK: - didTapEditButton
+    func testDidTapEditButton() {
+        viewController.product = Product()
+        viewController.product.barcode = "123456789"
+
+        viewController.didTapEditButton(UIButton())
```

Comment:
There is probably a better way to unit test against this.

You should technically send an event from editButton and not call the method. If I have in mind, it should be something like

`editButton.sendActions(for: .touchUpInside)`